active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
```

Comment:
I don't know if "Web Framework" should be capitalised unless we use this phrasing elsewhere.